doc/radosgw/pools.rst

```diff
@@ -55,3 +55,54 @@ the following pool entries use namespaces of the ``us-east.rgw.meta`` pool::
     "user_uid_pool": "us-east.rgw.meta:users.uid",
 
 .. _`Multisite Configuration`: ../multisite
+
+
+.. _create-ec-pool: 
+
+Create an Erasure-coded Pool
+----------------------------
+
+The process of creating an erasure-coded pool has two steps: 
+
+   #. Create a pool with the type "erasure". 
+   #. Associate the pool with the "rgw" application.
+
+.. warning::
+   The creation of an erasure-coded pool must happen before RGW deployment. After RGW starts up, it 
+   automatically creates pools of the "replicated" type unless pools of the "erasure" type already
+   exist.
+
+.. note::
+   This procedure has been designed to be as close to copy-and-pasteable as possible. However, you 
+   cannot use this procedure blindly. You must gather the following pieces of information and insert 
+   them into the correct places in the commands in this procedure:
+
+   #. The total number of placement groups in the pool.
+   #. The total number of placement groups for placement purposes (probably the same as the total number of placement groups in the pool).
+   #. The erasure code profile. See :ref:`erasure-code-profiles` and :ref:`osd-ec-profile-set` for more information about this.
+   #. The number of objects expected to be in the pool.
+   #. A preferred autoscale mode ("on", "off", and "warn" are the options).
+
+   If you are unsure of any of these pieces of information, read :ref:`createpool`.
+
+**Procedure for Creating an Erasure-coded Pool**
+
+#. Create a pool with type "erasure":
+
+   .. prompt:: bash $
+
+      ceph osd pool create default.rgw.buckets.data [{pg-num} [{pgp-num}]] erasure \ [erasure-code-profile] erasure-code [expected_num_objects] [--autoscale-mode=<on,off,warn>]
```

Comment:
the \ is not necessary here - it was used as a line break in the reference doc

Reply:
I've removed the \.